src/components/GetImageButton.js

```diff
@@ -1,24 +1,11 @@
 import React, {Component} from 'react';
 
-const API_KEY = 'X3aNQZj7BdSiNk8EUjNBImIKplVeWrVEfeVislMQ';
-
-export default class GetImageButton extends Component{
-
-fetchRoverImage(){
-
-  this.setState({camera: this.props.camera, rovers: this.props.rovers, sol: this.props.sol});
-    let cam = this.state.camera;
-    let rove = this.state.rovers;
-    let num = this.state.sol;
-
-    let imageUrl = `https://api.nasa.gov/mars-photos/api/v1/rovers/${rove}/photos?sol=${num}&camera=${cam}&api_key=${API_KEY}`;
-};
-
-  render(){
```

Comment:
Because we are exporting a class, we don't need a render method -- indeed, functions don't have methods.